package.json

```diff
@@ -5,12 +5,12 @@
   "license": "GPLv3",
   "scripts": {
     "build": "babel src --extensions .js,.ts,.tsx --out-dir dist",
-    "lint": "eslint --ignore-path .gitignore --ext .js,.ts,.tsx .",
+    "lint": "eslint --ignore-path .gitignore --ext .js,.ts,.tsx",
```

Comment:
Unfortunately this means that if we want to run `npm run lint`, we'd have to pass `-- .` (as you do in the `validate` script) which would be a pain.

I understand the trade-off you're proposing here though. Makes sense, but I think I prefer it the way it is. Thanks anyway!